src/components/animated/_snake.less

```diff
@@ -46,10 +50,23 @@
         height 150ms 300ms cubic-bezier(0.07, 0.62, 0.61, 1);
       border-color: #333;
     }
+
+    [data-theme="dark"] &,
+    &.dark-mode {
+      &:before {
```

Comment:
Change the selector to `&:before, &:after` and remove the rule for `&:after` below.